### DISCUSSION TITLE
Adjust default key counter for legacy skins to match stable

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#25708] [Ideas] [answered]
When you import a legacy skin into osu!(lazer), the interface will look something like this, with the key counter being in the bottom right:

![osu_2023-12-08_21-22-14](https://github.com/ppy/osu/assets/74088315/4777846d-6c0e-487a-b806-c3609234d7d3)

I propose that it should look something like this instead, by utilizing the new argon key counter and putting it in the center right of the screen in a way that is very similar to stable:

![osu_2023-12-08_21-28-49](https://github.com/ppy/osu/assets/74088315/e689e62a-f225-4c0a-bcd7-80f76f2d6ab0)

Given that it seems like the default interface for legacy skins is supposed to match stable as much as possible, I think that this placement for the key counter makes more sense as it is very similar to the key counter in stable.

**Comment by hrfarmer** [comment]
Didn't even notice someone already made this discussion, but I made a branch to do exactly this. If someone on the osu! team is okay with this I can make a pr.

https://github.com/hrfarmer/osu/commit/377db189491aa4e19fef0091209076abbd0d10c9

https://github.com/ppy/osu/assets/58487401/47a6fd5b-5c13-4307-9fa4-36989bf479a1

**Reply by bdach** [reply]
If anything the legacy assets should be getting used and not the argon key counter. As in the legacy counter should be actually implemented.

**Reply by Joehuu** [reply, marked as answer]
Legacy key counter added in https://github.com/ppy/osu/pull/29027.
